static/js/redux/state/saving_timetable_reducer.jsx

```diff
@@ -48,7 +48,7 @@ const savingTimetable = (state = initState, action) => {
     case ActionTypes.ADD_CUSTOM_SLOT:
     case ActionTypes.UPDATE_CUSTOM_SLOT:
     case ActionTypes.REMOVE_CUSTOM_SLOT:
-    case ActionTypes.CHANGE_ACTIVE_TIMETABLE:
+    case 'global/changeActiveTimetable':
```

Comment:
Just curious, is there a reason why these are `'global/...'` strings and aren't directly referenced by importing the action?

Reply:
I think we can't directly use redux toolkit actions in traditional non-rtk reducers, so I think I have to use that name of the action which is the string. I will change it back to importing the action when I refactor this reducer

Reply:
I see, that makes sense